frontend/src/components/BossesID.jsx

```diff
@@ -0,0 +1,3 @@
+const BossesID = [639, 320, 141, 532, 655]; // Stromtrooper, Heat-Wave, Bullseye, Pyro, Thanos
+
+export default BossesID;
```

Comment:
it's not a component... putting it outside a component means putting it outside the component function. you can keep it in the same file